### PR TITLE
added tasks

### PR DESCRIPTION
**Pull request by ahmed-khatri**
added taks

### CODE REVIEW

**Comment by ahmed-khatri** [comment]
hello

File: readme.md
Comment (AhmedKhatri510) [inline readme.md:1]: change the line

Comment (AhmedKhatri510) [inline readme.md:1]: update the line

Comment (AhmedKhatri510) [inline readme.md:1]: correct the spellings

**Review by AhmedKhatri510 — APPROVED** [review]

File: readme.md
Comment (AhmedKhatri510) [inline readme.md:1]: sdaf

**Review by AhmedKhatri510 — APPROVED** [review]

File: readme.md
Comment (AhmedKhatri510) [inline readme.md:1]: correct the spellings

**Review by SaurabhPatil240 — APPROVED** [review]

**Review by AhmedKhatri510 — COMMENTED** [review]
asfasdfsdfsd

**Review by AhmedKhatri510 — APPROVED** [review]

**Review by AhmedKhatri510 — CHANGES_REQUESTED** [review]
sdfsdfsdfsdf

**Pull request merged**